.github/actions/authenticate-aws/action.yaml

```diff
@@ -1,9 +1,14 @@
 name: AuthenticateAWS
```

Comment:
We should just drop this action. IMO, this composite action isn't giving us anything different than calling the `aws-actions/configure-aws-credentials` action